ckanext/custom_schema/tests/test_helpers.py

```diff
@@ -0,0 +1,75 @@
+import ckan.lib.search
+from ckantoolkit.tests import helpers, factories
+
+from ckanext.custom_schema.helpers import (
+    get_group_list,
+    get_selected_group
+)
+
+
+class TestGetGroupList(object):
+    @classmethod
+    def setup(self):
+        helpers.reset_db()
+        ckan.lib.search.clear_all()
+
+    @classmethod
+    def teardown(self):
+        helpers.reset_db()
+        ckan.lib.search.clear_all()
+
+    def test_group_list_no_results(self):
+        assert get_group_list() == []
+
+    def test_group_list_with_results(self):
+        group1 = factories.Group(name='water')
+        group2 = factories.Group(name='transportation')
+        group3 = factories.Group(name='economy')
+        group_from_helpers = get_group_list()
+
+        assert group_from_helpers[0]['name'] == 'economy'
+        assert group_from_helpers[1]['name'] == 'transportation'
+        assert group_from_helpers[2]['name'] == 'water'
```

Comment:
Can `assert_list_equal` be used instead?

Reply:
To use `assert_list_equal` we would need to import the python package `unittest`. I think it's not necessary if we're just asserting it a few times.

Reply:
I am not really sure about ordering here.
In the future, we can have a problem with a failed test because of ordering in the list for any reason. I would like to compare sorted lists here because the order is not critical for us.

Reply:
The results from get_group_list() is sorted by title so it should not be a problem. In fact if the results are in any order order the test should fail.